src/messaging/ExchangeContext.cpp

```diff
@@ -689,5 +695,25 @@ void ExchangeContext::ExchangeSessionHolder::GrabExpiredSession(const SessionHan
     GrabUnchecked(session);
 }
 
+bool ExchangeContext::IsPeerActive()
+{
+    // We can never assume the peer is active if we are sending a group message
```

Comment:
Honestly, the concept just makes no sense, in that there is no "peer".

But there is also no MRP for group messages, so in practice I would expect this to never be called for a group context....

Reply:
I'll remove the group check. Group context should never end up calling this function like you said.